airflow/config_templates/default_airflow.cfg

```diff
@@ -510,6 +510,11 @@ authenticate = False
 # DAGs submitted manually in the web UI or with trigger_dag will still run.
 use_job_schedule = True
 
+# If set to true, backfill will create dagrun in reverse order if the dag
+# doesn't depend on past. Default is false which it will still create dagrun
+# from start date to end date.
+backfill_dagrun_order_reverse = False
```

Comment:
Saying "reverse" relies on the operator knowing what the default order is, which isn't explained anywhere else - it would be clearer if we called this `backfill_from_most_recent_to_oldest = False` or some variation of that. Yes it's explained in the comment but we can make the env var or config name make more sense,